components/apim/main.tf

```diff
@@ -14,11 +14,13 @@ module "api-mgmt" {
   virtual_network_resource_group = local.vnet_rg
   virtual_network_name           = local.vnet_name
   environment                    = var.env
+  env                            = var.env
```

Comment:
best to change here
https://github.com/hmcts/cnp-module-api-mgmt-private/blob/main/app-insights.tf#L4  with `local.environment`